projects/kubernetes-sigs/image-builder/Makefile

```diff
@@ -106,9 +130,14 @@ deps-ova: MAKEFLAGS=
 deps-ova: $(GIT_PATCH_TARGET)
 	$(MAKE) -C $(IMAGE_BUILDER_DIR) deps-ova
 
+.PHONY: deps-raw
+deps-raw: MAKEFLAGS=
+deps-raw: $(GIT_PATCH_TARGET)
```

Comment:
Since these deps ones are all just pass throughs i bet we could combine all these as well use a wildcard target?

Reply:
Done